diff --git a/src/core/include/units/quantity.h b/src/core/include/units/quantity.h
index 306e460b..aebeb89c 100644
--- a/src/core/include/units/quantity.h
+++ b/src/core/include/units/quantity.h
@@ -1,4 +1,3 @@
-
 // The MIT License (MIT)
 //
 // Copyright (c) 2018 Mateusz Pusz
@@ -443,7 +442,7 @@ public:
 };
 
 // CTAD
-#if !UNITS_COMP_CLANG || UNITS_COMP_CLANG > 17
+#if 0
 template<typename D, typename U, typename Rep>
 explicit(false) quantity(Rep&&) -> quantity<D, U, Rep>;
 #endif
